docs/content/configuration/creating_storageclass_topology_aware.md

```diff
@@ -14,28 +14,26 @@ The various `by_management_id` parameters are chosen within the following hierar
 1. From within the `by_management_id` parameter, per system (if specified).
 2. Outside of the parameter, as a cross-system default (if not specified within the `by_management_id` parameter for the relevant `management-id`).
 
-  
-  ```
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: demo-storageclass-config-secret
-provisioner: block.csi.ibm.com
-volumeBindingMode: WaitForFirstConsumer
-parameters:
-  # non-csi.storage.k8s.io parameters may be specified in by_management_id per system and/or outside by_management_id as the cross-system default.
-
-  by_management_id: '{"demo-management-id-1":{"pool":"demo-pool-1","SpaceEfficiency":"deduplicated","volume_name_prefix":"demo-prefix-1"},
-                      "demo-management-id-2":{"pool":"demo-pool-2","volume_name_prefix":"demo-prefix-2"}}'  # Optional.
-  pool: demo-pool
-  SpaceEfficiency: thin            # Optional.
-  volume_name_prefix: demo-prefix  # Optional.
-
-  csi.storage.k8s.io/fstype: xfs   # Optional. Values ext4/xfs. The default is ext4.
-  csi.storage.k8s.io/secret-name: demo-config-secret
-  csi.storage.k8s.io/secret-namespace: default
-allowVolumeExpansion: true
-  ```
+        kind: StorageClass
+        apiVersion: storage.k8s.io/v1
+        metadata:
+          name: demo-storageclass-config-secret
+        provisioner: block.csi.ibm.com
+        volumeBindingMode: WaitForFirstConsumer
+        parameters:
+          # non-csi.storage.k8s.io parameters may be specified in by_management_id per system and/or outside by_management_id as the cross-system default.
+
+          by_management_id: '{"demo-management-id-1":{"pool":"demo-pool-1","SpaceEfficiency":"deduplicated","volume_name_prefix":"demo-prefix-1"},
```

Comment:
@ArbelNathan I think we should avoid deprecated values in [our examples](https://github.com/IBM/ibm-block-csi-driver/blob/312a4aafe9f95d3cebabeb6a72e2d283c925a503/deploy/kubernetes/examples/demo-volumesnapshotclass-config-secret.yaml#L10)

Reply:
right, I think it's too late for driver changes now but we can change here  `deduplicated` to `dedup_compressed`, that way it will have the same meaning at least.

Reply:
done (there were 2 examples and i updated them both)

Reply:
it should be `dedup_compressed` ([not `dedup_compressed_`](https://github.com/IBM/ibm-block-csi-driver/blob/5140b8ac69b78716d1783844b35f3dedaab5a6a9/docs/content/configuration/creating_volumereplicationclass_topology_aware.md))

Reply:
why too late for driver changes now? and if so, do we have a ticket for it?

Reply:
if you think we have time IDC to do it..

Reply:
I agree it can wait for 1.9.0 if need be, I just don't want to completely miss it

Reply:
https://github.com/IBM/ibm-block-csi-driver/pull/421

Reply:
thanks!

Reply:
ok, so just [this](https://github.com/IBM/ibm-block-csi-driver/pull/415#discussion_r769414312)

Reply:
done (sorry for the typo)